### PR TITLE
Rust 1.3 announcement

### PR DESCRIPTION
**Pull request by aturon**
r? @brson @steveklabnik

### CODE REVIEW

File: _posts/2015-09-17-Rust-1.3.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+---
+layout: post
+title: "Announcing Rust 1.3"
+author: The Rust Core Team
+---
+
+The gear keeps turning: we're releasing Rust 1.3 stable today! As always, read
```
Comment (steveklabnik) [inline _posts/2015-09-17-Rust-1.3.md]: "gears keep turning" ?

Comment (mrmonday) [inline _posts/2015-09-17-Rust-1.3.md]: Whilst I'd normally agree, I think it's a pun (Rust logo being a gear and all...)

File: _posts/2015-09-17-Rust-1.3.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+This is our first release shipping with the
+[Rustonomicon](https://doc.rust-lang.org/nightly/nomicon/), a new book covering
+"The Dark Arts of Advanced and Unsafe Rust Programming". While it's still in
+draft form, this book already provides deep coverage of some of Rust's darker
+corners.
+
+On the library front, we saw a fair amount of API stabilization, including the
+new `Duration` API and enhancements to `Error` and `Hash`/`Hasher`. We expect to
+see further growth of the `std::time` module in the 1.5 timeframe.
```
Comment (mrmonday) [inline _posts/2015-09-17-Rust-1.3.md]: Should this be `1.4`?

Comment (aturon) [inline _posts/2015-09-17-Rust-1.3.md]: Nope, hasn't landed yet.

File: _posts/2015-09-17-Rust-1.3.md (hunk 1 above)
Comment (aturon) [inline _posts/2015-09-17-Rust-1.3.md]: Yeah, that was the intended reference. Maybe too subtle?

Comment (steveklabnik) [inline _posts/2015-09-17-Rust-1.3.md]: if only we had some tooling to statically verify that a reference points to the correct referent
trollface

File: _posts/2015-09-17-Rust-1.3.md (hunk 2 above)
Comment (mrmonday) [inline _posts/2015-09-17-Rust-1.3.md]: OK, was just checking :)

**Pull request merged**